recipes-devtools/partition-utils/qcom-gen-partition-bins_1.0.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Generate and deploy the machine specific partition binaries"
+LICENSE = "BSD-3-Clause-Clear"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/${LICENSE};md5=7a434440b651f4a472ca93716d01033a"
+
+DEPENDS += "qcom-ptool-native"
+
+inherit deploy python3native
+
+do_configure[noexec] = "1"
+do_install[noexec] = "1"
+do_compile[depends] += "virtual/partconf:do_deploy"
+
+do_compile() {
+    ${PYTHON} ${STAGING_BINDIR_NATIVE}/ptool.py -x ${DEPLOY_DIR_IMAGE}/partition.xml
```

Comment:
this is not correct. do_compile() cannot copy to DEPLOY_DIR_IMAGE. We should create partition.xml in S or B, and then install to DEPLOYDIR from do_deploy.

Reply:
oh. sorry. partition.xml is generated from partconf:do_deploy, it's an input file here. so that is correct.

Reply:
same here, call just the tool works
```patch
-    ${PYTHON} ${STAGING_BINDIR_NATIVE}/ptool.py -x ${DEPLOY_DIR_IMAGE}/partition.xml
+    ptool.py -x ${DEPLOY_DIR_IMAGE}/partition.xml
```